src/sdg/scss/components/_tabs.scss

```diff
@@ -100,6 +100,32 @@
     line-height: 1.5;
     color: #212121;
 
+    &--warning {
```

Comment:
I don't necessarily like the way this is set up. Is, this a tab component, a table component, or a notification component?

The (BEM) methodology we use indentifies 3 layers of naming.

Block - A standalone, reusable component). (e.g. `.tabs`).
Element - A dependency of a block, without any meaning on it's own (e.g. `.tabs__tab`).
Modifier A theme or a state of either a block or an element (e.g. `.tabs__tab--active).

__Please not that a modifier should always be combined with the block or element classname__

So do:

```html
<div class="tabs__tab tabs__tab--active"></div>
```

And dont do:

```html
<div class="tabs__tab--active"></div>
```

The idea of BEM is that everything is as scoped (isolated) and reusable as possible and is able to express some minimal state using simple class names. 

In this case, `tabs` is a good example of a BEM block, however  `tabs__table` is not a good example of an element. A better solution would be to create a new file `table` which contains the styling for a table without any dependency on tabs (as we would like to be able to use a table outside the context of a tab). A better HTML tree would look something like:

```html
<div class="tabs">
  (...)
  <div class="tabs_tab tabs__tab--active">
    <table class="table>
      (...)
    </table>
  </div>
</div>
```

If a tab would override de styling of a table, a better solution would be something like `.tabs .table`.